test/pass/arith.sest

```diff
@@ -0,0 +1,7 @@
+module Arith = struct
+
+  val main(_) =
+    let 13 = 3 * 4 + 1 in
```

Comment:
```suggestion
    let 13 = 1 + 3 * 4 in
```

Reply:
Thank you! I simply added the suggested test case at 0a00ef3.